config/samples/ramendr_v1alpha1_volumereplicationgroup.yaml

```diff
@@ -4,19 +4,15 @@ metadata:
   name: volumereplicationgroup-sample-1
 spec:
   # Add fields here
-  affinedCluster: us-central
-  applicationLabels:
-    matchExpressions:
-    - key: apptype
-      operator: NotIn
-      values:
-      - stateless
-    - {key: backend, operator: In, values: [ceph]}
+  pvcSelector:
+  #  matchExpressions:
```

Comment:
(nit) just remove these, instead of comments?

Reply:
Being new to the details of label selectors, I was perhaps pleasantly surprised by the versatility of label selectors and must've wanted to highlight it when I wrote those.  Given that match expressions are supported and seem more powerful than matchLabels, I thought of at least leaving matchExpressions as a comment for other Ramen developers to appreciate the possibilities behind label selectors and not assume a simple matchLabel like I did.  Looks like the comments are sore to the eye. I will remove it.